src/CMakeLists.txt

```diff
@@ -30,6 +30,11 @@ include_directories(
     ${CMAKE_CURRENT_BINARY_DIR}
 )
 
+file(GLOB_RECURSE CLANG_FILES *.cc *.h)
+add_custom_target(clang-format
+    COMMAND /usr/bin/clang-format -style=file --verbose -i ${CLANG_FILES}
+)
+
```

Comment:
It would be also nice to have another target to check the formatting. Something like this:
```
file(GLOB_RECURSE CLANG_FILES *.cc *.h)
add_custom_target(clang-format-check
    COMMAND /usr/bin/clang-format -style=file --dry-run -Werror --verbose -i ${CLANG_FILES}
)
```
A similar "check" option exists in the command line interface for compiling Qserv:
```
qserv build --clang-format [CHECK|REFORMAT|OFF]
```